src/test/java/bio/terra/service/auth/iam/sam/SamIamTest.java

```diff
@@ -82,44 +76,49 @@ public class SamIamTest {
   @Mock private StatusApi samStatusApi;
   @Mock private GoogleApi samGoogleApi;
   @Mock private UsersApi samUsersApi;
-  @Mock private TermsOfServiceApi samTosApi;
   @Mock private GroupApi samGroupApi;
 
   private SamIam samIam;
   private static final String ADMIN_EMAIL = "samAdminGroupEmail@a.com";
   private static final AuthenticatedUserRequest TEST_USER =
       AuthenticationFixtures.randomUserRequest();
 
-  @Before
-  public void setUp() throws Exception {
+  @BeforeEach
+  void setUp() throws Exception {
     samIam = new SamIam(samConfig, configurationService, samApiService);
```

Comment:
I didn't mention it before, but another thing I think we should stop doing in our tests is mocking data objects. In this case, both `samConfig` and `configurationService` are really data objects -- they don't contain any mutable state and don't connect to any live services. So instead of mocking `samConfig` it could be declared as
```
  private final SamConfiguration samConfig = new SamConfiguration();
```
And this method would be:
```
  void setUp() {
    samConfig.setAdminsGroupEmail(ADMIN_EMAIL);
    samIam =
        new SamIam(
            samConfig,
            new ConfigurationService(
                samConfig, null, new GoogleResourceConfiguration(), new ApplicationConfiguration()),
            samApiService);
  }
```